lib/extensions/ar_taggable.rb

```diff
@@ -4,6 +4,8 @@ def self.acts_as_miq_taggable
       has_many :taggings, :as => :taggable, :dependent => :destroy
       has_many :tags, :through => :taggings
       include ActsAsTaggable
+
+      after_create :apply_user_tags
```

Comment:
@lpichler let chat when you are here, e.g. in batch saving this would not work. For normal saving, this would fire up for every model that is taggable? So there is significant perf impact.

Reply:
I removed after created, only this PR is adding only method.